tools/flavor-convert/new_flavorpart.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+package main
+
+import "fmt"
+
+type PcrLogs struct {
```

Comment:
Please note that since the new flavor part changes are not present in intel-secl master branch, we have pushed the flavor creation/verification changes in our own repo and replaced the intel-secl path with this repo path in go mod file. Once the flavor creation/verification changes are approved and merged with master branch, we will remove this path from go mod file.